Added a warning in the ambient getting started guide (#13757)
istio/istio.io: content/en/docs/setup/platform-setup/kind/index.md

    Deleting cluster "istio-testing" ...
    {{< /text >}}

## Setup MetalLB for kind

> kind does not have any inbuilt way to provide IP addresses to your `Loadbalancer` type services, to ensure you get IP's you can install [MetalLB](https://kind.sigs.k8s.io/docs/user/loadbalancer/).

Reviewer: ```suggestion
Kind does not have any built-in way to provide IP addresses to your `Loadbalancer` service types. To ensure IP address assignments to Gateway Services, please consult [this guide](https://kind.sigs.k8s.io/docs/user/loadbalancer/) for more information.
```

Reviewer: Also need a blank line between this line and the header on line 74. This should fail linting

Author: Ah yes it did fail but couldn't figure out why, saw the output again and I see it now
```
./en/docs/setup/platform-setup/kind/index.md:73: MD022 Headers should be surrounded by blank lines
```

    Deleting cluster "istio-testing" ...
    {{< /text >}}

## Setup MetalLB for kind

> Kind does not have any built-in way to provide IP addresses to your `Loadbalancer` service types, to ensure IP address assignments to `Gateway` Services please consult [this guide](https://kind.sigs.k8s.io/docs/user/loadbalancer/) for more information.

Reviewer 2: kind does not get capitalized at the beginning of sentences as noted on their doc pages.
```suggestion
kind does not have any built-in way to provide IP addresses to your `Loadbalancer` service types, to ensure IP address assignments to `Gateway` Services please consult [this guide](https://kind.sigs.k8s.io/docs/user/loadbalancer/) for more information.
```

Author: Missed the notif, will make changes

> kind does not have any built-in way to provide IP addresses to your `Loadbalancer` service types, to ensure IP address assignments to `Gateway` Services please consult [this guide](https://kind.sigs.k8s.io/docs/user/loadbalancer/) for more information.

Reviewer 2: ```suggestion
kind does not have any built-in way to provide IP addresses to your `Loadbalancer` service types. To ensure IP address assignments to `Gateway` Services please consult [this guide](https://kind.sigs.k8s.io/docs/user/loadbalancer/) for more information.
```

## Setup Dashboard UI for kind

> Kind does not have a built-in Dashboard UI like minikube. But you can still setup Dashboard, a web based Kubernetes UI, to view your cluster.

Reviewer 2: kind is correctly lowercase.

> kind does not have a built-in Dashboard UI like minikube. But you can still setup Dashboard, a web based Kubernetes UI, to view your cluster.

Reviewer: ```suggestion
kind does not have a built-in Dashboard UI like minikube. But you can still setup Dashboard, a web-based Kubernetes UI, to view your cluster.
```

Author: Thanks I also managed to fix the ordering I mentioned earlier. Let me push this too! thank you
